modin/experimental/xgboost/utils.py

```diff
@@ -40,7 +40,8 @@ def __init__(self, num_workers: int, host_ip):
         self.rabit_tracker = xgb.RabitTracker(hostIP=host_ip, nslave=self._num_workers)
 
     def __enter__(self):
-        """Entry point of manager.
+        """
+        Entry point of manager.
```

Comment:
I think we agreed on Numpy standard where docstring has to begin right after quotes, not on a new line...
here and in other places

Reply:
This is how it was originally, but it was changed due to exceptions from script https://github.com/modin-project/modin/issues/2934#issuecomment-816125767. @anmyachev what style are we have to use?

Reply:
It's `GL01` restriction: https://github.com/modin-project/modin/pull/2968#discussion_r612620623

If we agree with this check, then we will leave it that way.